forge-gui/res/cardsfolder/r/relentless_skaabs.txt

```diff
@@ -3,5 +3,5 @@ ManaCost:3 U U
 Types:Creature Zombie
 PT:4/4
 K:Undying
-A:SP$ PermanentCreature | Cost$ 3 U U ExileFromGrave<1/Creature>
+A:SP$ PermanentCreature | Cost$ 3 U U ExileFromGrave<1/Creature> | SpellDescription$ As an additional cost to cast this spell, exile a creature card from your graveyard.
```

Comment:
why are we adding it if the autogeneration works fine?

Reply:
The automatic text doesn't show up in full on the Prompt, and I'd rather err on the side of clarity, especially in those admittedly rarer cases where mana costs aren't paid and casting is mandatory.
![image](https://github.com/user-attachments/assets/82ff6dc5-5520-4c03-ab9d-1359465f800e)

Reply:
Hmn, can't say I like reproducing this (especially in the script)
I'll try and check if it can be included from the code instead

Reply:
Fair. I'll be checking the gnarlier variants for issues and address what I can.

Reply:
check #6233
this should let you avoid these changes?

Reply:
So far, I feel like the issues I'm coming across are an extension of what I've noted here, https://github.com/Card-Forge/forge/issues/5916 , that time for non-permanent spells and activated abilities, 
![image](https://github.com/user-attachments/assets/0f7f87d4-0872-4c23-8682-14c8b7ce4424).

Reply:
> check #6233 this should let you avoid these changes?

To be clear, the above was done without reference to your fix, though I'll be sure to test it as soon as possible.

Reply:
Preliminary fix testing done: works fine indeed. While reverting the simpler additional cost variants should be easy enough, the more complex variants do need a closer look.
![image](https://github.com/user-attachments/assets/489f831f-05d6-41ec-885e-a9a0dbc9cba1)

Reply:
ok, I've changed it so can you can still use the override with SpellDescription if there's a really exotic case
however usually the changes should rather be made/fixed inside the part description handling imo
it's also worth mentioning a newer approach without manual spell definition (see #2472) could fix some more cleanly (though your desired full text in prompt might not happen then)

Reply:
Fair. I'll do my best to do as you recommend 👍

Reply:
As a confounding contrast to [Redemptor Dreadnought](https://scryfall.com/card/40k/164/redemptor-dreadnought) ( [card script](https://github.com/Card-Forge/forge/blob/master/forge-gui/res/cardsfolder/r/redemptor_dreadnought.txt) ), I came across [Graven Archfiend](https://scryfall.com/card/ysnc/7/graven-archfiend) ( [card script](https://github.com/Card-Forge/forge/blob/master/forge-gui/res/cardsfolder/g/graven_archfiend.txt) ), which uses `S:Mode$ OptionalCost` for its optional additional cost instead. The issue with the Dreadnought is the text on the Prompt is fairly off base, while the Archfiend works like kicker does. 

![image](https://github.com/user-attachments/assets/a275074b-09b1-4e3b-bd88-1e8f6e1bede4)

So, should the optional additional costs be changed to static abilities or is there a reason why `A:SP$ PermanentCreature` is the better choice the Dreadnought?

Reply:
_Redemptor Dreadnought_ was written at a time OptionalCost static didn't exist yet
it could be updated now to improve the text (while I would prefer if it could still be autogenerated in that case that's something for later)
I believe some cards like _Gorex_ are still impossible in that fashion though

PS:
It looked to me like the remaining parts of this run all had at least 1 script that was changed in this way, so I was going to wait until we have an update approach that can be applied on all

Reply:
Duly noted 👍

Reply:
Oh, I see the Dreadnought would also display its flavor words if it gets changed :)

Reply:
Uhm anyway, I'm just gonna try and rephrase since I feel I might not have been clear? 😅
The cards don't need to be updated identical as of yet. Just try and get rid of any "duplicated" text that my PR would display now instead - any bad phrasing that still occurs should really be fixed in code since it would still display wrong in card detail imo (but I leave that to your discretion if it doesn't collide with anything else).

Then I can merge all together...

Reply:
I'm certainly leaving the second paragraph on the Prompt for later.